docs/reference/changelog.rst

```diff
@@ -2,6 +2,11 @@
 Changelog
 *********
 
+X.Y.Z (2025-MM-DD)
```

Comment:
We can call this 0.6 since it's not really a bug patch but a behaviour change.

Reply:
Versioned!